test.js

```diff
@@ -90,7 +91,8 @@ const names = {
 	lz: ['fixture.tar'],
 	Z: ['fixture.tar'],
 	mkv: ['fixture', 'fixture2'],
-	mxf: ['fixture']
+	mxf: ['fixture'],
+	'3gp': ['fixture']
```

Comment:
This isn't needed since it'll test `fixture.3gp` by default. These are only needed for fixtures that needs special handling.